server/src/test/java/org/elasticsearch/common/util/concurrent/EsExecutorsTests.java

```diff
@@ -787,13 +793,11 @@ public void testScalingWithEmptyCoreAndKeepAlive() {
     }
 
     public void testScalingWithEmptyCoreAndLargerMaxSize() {
-        // TODO currently the reproduction of the starvation bug does not work if max pool size > 1
-        // https://github.com/elastic/elasticsearch/issues/124867
-        testScalingWithEmptyCore(
+        testScalingWithEmptyCoreAndMaxMultipleThreads(
             EsExecutors.newScaling(
                 getTestName(),
                 0,
-                between(2, 5),
+                2,// between(2, 5),
```

Comment:
leftover from development?

```suggestion
                between(2, 5),
```

Reply:
good catch, thx 👍